src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -413,13 +411,6 @@ public override void DefaultVisit(Symbol symbol)
 
                 bool reportDiagnosticsForCurrentTrivia = trivia.SyntaxTree.ReportDocumentationCommentDiagnostics();
 
-                // If we're writing XML or we need to report diagnostics (either in this particular piece of trivia,
-                // or concerning undocumented [type] parameters), then we need to process this trivia node.
-                if (!(haveWriter || reportDiagnosticsForCurrentTrivia || reportParameterOrTypeParameterDiagnostics))
```

Comment:
can we remove reportParameterOrTypeParameterDiagnostics from the parameter list now?

Reply:
Used on line 337.

---
In reply to: [184566077](https://github.com/dotnet/roslyn/pull/26425#discussion_r184566077) [](ancestors = 184566077)